cms/djangoapps/contentstore/features/discussion-editor.feature

```diff
@@ -5,7 +5,7 @@ Feature: CMS.Discussion Component Editor
   Scenario: User can view discussion component metadata
     Given I have created a Discussion Tag
     And I edit the component
-    Then I see three alphabetized settings and their expected values
+    Then I see three settings and their expected values
```

Comment:
Yes, the settings are no longer alphabetized.  They're in field order, as specified by the [DiscussionXBlock class](https://github.com/edx/edx-platform/pull/12582/files#diff-480a9bebc712a4e39a33ab93f415017aR19).

The new order is more logical from a user perspective too.  It puts the Display Name first, then Category and Subcategory, as verified by [the updated test](https://github.com/edx/edx-platform/pull/12582/files#diff-8823bc982a355f230d05625874ec0ce6R16).